ideascube/serveradmin/catalog.py

```diff
@@ -351,18 +351,31 @@ def _get_package(self, id, source):
         except KeyError:
             raise InvalidPackageType(type)
 
-    def _get_packages(self, id_patterns, source, fail_if_no_match=True):
+    def _get_packages(
+        self, id_patterns, source,
+        fail_if_no_match=True, fail_if_invalid_package=True):
```

Comment:
`fail_if_invalid_package` => `fail_if_invalid` given that we are talking about packages?

BTW, I usually think that having two boolean kwargs on the same method is a refactor needed alert.